src/pages/Add-items/Add-items.js

```diff
@@ -39,6 +41,10 @@ export const AddItems = () => {
     showModal();
   };
 
+  if (!product.token) {
+    return <Redirection />;
```

Comment:
¿Qué opinan de verificar además de la existencia del token, que el token se valido?

```suggestion
  if (!isValidToken(product.token)) {
    return <Redirection />;
    }
```

Reply:
Súper. Está hecho